### PR TITLE
Integrate Flask-JWT with Flask-Login

### PR DESCRIPTION
**Pull request by rasendubi**
This is the first step needed for integration of Flask-User (Flask-User
uses Flask-Login), as we need to keep JWT authentication keep working.

### CODE REVIEW

File: heutagogy/auth.py [hunk 1]
```diff
@@ -14,7 +15,20 @@ def authenticate(username, password):
 
 
 def identity(payload):
-    return get_user(payload['identity'])
+    user = get_user(payload['identity'])
+    if user:
+        # this field is needed for Flask-Login
+        user.is_authenticated = True
+    return user
```
Comment (drets) [inline heutagogy/auth.py:22]: Is it OK if we return `None` here?

**Comment by drets** [comment]
Could you remind me why we want `Flask-User`?

**Comment by rasendubi** [comment]
Flask-User handles user registration, authorization, confirmation and many other things I don't want to implement myself. :smirk:

File: heutagogy/auth.py (hunk 1 above)
Comment (rasendubi) [reply heutagogy/auth.py:22]: Sure: https://github.com/mattupstate/flask-jwt/blob/master/flask_jwt/__init__.py#L162-L165

**Pull request merged**